server/routes.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-router.get("/", (req, res) => {
+router.get("/", (_req, res) => {
```

Comment:
Why is this `_req, res` and not `_req, _res`?

Reply:
_is a short hand for unused var

Reply:
`_` as a prefix (e.g. `_myVariable`) denotes a private variable.
General consensus for an unused variable is just to use `_`. So this would be:
`(_, res)`